### ISSUE TITLE
Agario | www.agar.io

### ISSUE THREAD

**Issue by samirsask** [PreMiD/Presences#947]
**Prerequisites and essential questions** 
* [x] Is it a popular site?
* [x] Is the website older than 2 months? 
* [ ] Is the site locked to a specific country/region?
* [ ] Is the site a paid service? (e.g. Netflix, Hulu)
* [ ] Does the website feature NSFW content? (e.g. porn, etc...)
* [ ] Are you a donator/patron?
* [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority?

**What's your Discord username?** 
  Sask#0002 

**What's the name of the service?**  
 www.agar.io | Agario 

**What should the Presence display?** 
 "Playing Agar.io". It's a famous miniclip game , it would be nice to have the duration of the game and the score. 

**If possible, please provide a logo for the service (512x512 minimum)** https://imgur.com/x2QZCXF.png

**Comment by Bas950** [comment]
Its seems like due to site limitations I am not able to get any data while playing, only after.
